posts/2020-06-08-announcing-the-moving-from-scala-2-to-scala-3-course.adoc

```diff
@@ -0,0 +1,40 @@
+
+Pretty exciting stuff I still think, and I'm sure that this approach will be adopted in a future version of the Akka toolkit as an alternative to the rather convoluted way of dealing with responses to messages in Akka Typed (something which is necessary because of some limitations of the Scala 2 compiler).
+
+So, with that behind me, I was wondering about what to do next... Now, I've always been very keen on learning new stuff, but even more on sharing knowledge with others. I've taught many technical courses and [co-]developed different courses on Scala and Akka while at Lightbend. So, a little less than two months ago, I decided to start working on a new course that would target existing Scala 2 developers and that would help them in the move to Scala 3. The general idea I started from was to take an existing Scala 2 application, select a number of key features in Dotty, and apply these to the application in a step-by-step fashion. The end goal was to show how these features and improve and reduce the complexity in the original code.
+
+When I launched the idea about this new course at Lunatech, I got two enthusiastic colleagues, _Neil Benn_ and _Robin Raju_, to join me in the effort. So, after about 2 months in elapsed time, we delivered the first edition of the two-day _"Moving from Scala 2 to Scala 3"_ course to a bunch of colleagues.
+
+Between the start of the project and the first delivery, the alpha version of the course was kindly and thoroughly reviewed by members of the Scala Center. We received a lot of very useful feedback which resulted in a series of corresponding updates to the course materials. Speaking of which: We have a series of 11 exercises, a course slide deck with approximately 140 slides, and a collection of code-snippets that can be used to demonstrate various features of the Dotty language treated in the course.
+
+Very important to note is that all these materials are open-sourced! You can find them in https://github.com/lunatech-labs/lunatech-scala-2-to-scala3-course[this public repository] on Github. This means that you can benefit from this at no extra cost but the effort you put into studying the course materials! _(Of course, should you wish to have this course taught to you in-class, ping us —although that will not be free of charge)_.
```

Comment:
I'd recommend the changing the last part about "-although that will not be free of charge". I think that's sort of implied, and just saying something about  "should you wish to have this course taught to you in-class, feel free to reach out to us and we're happy to talk!". I think this reads a bit better.

Reply:
Agreed and a great suggestion!